pkg/encoding/property.go

```diff
@@ -0,0 +1,1853 @@
+package encoding
+
+type BACnetNetworkType int
```

Comment:
Since this package is called `bacnet` why not `NetworkType`

Reply:
we already have it declared in the same package. also the package is "encoding"

Reply:
This is an example https://github.com/SammyOina/bacnet/blob/d2c5be983822d6adeaa61b7eff753e49d1ae477e/example/readProperty/readProperty.go#L16

Since it is bacnet it better to name it as
```go
destination := bacnet.NewAddress(0, nil, "127.0.0.6:47809", &netType)
```
Instead of 
```go
destination := bacnet.NewBACnetAddress(0, nil, "127.0.0.6:47809", &netType)
```